### PR TITLE
Add a /ping route for templates to ping pong

### PR DESCRIPTION
**Pull request by assefamaru**
This PR adds a /ping route. Also adds MySQL configuration instead of the default sqlite.

### CODE REVIEW

File: .sandbox/manifest.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
   build:
-    cmd: 'pip3 install django'
+    cmd: ./sandbox.sh
 
 daemons:
   app:
     run:
-      cmd: 'python3 manage.py runserver'
\ No newline at end of file
+      cmd: python3 manage.py runserver 0:3000
```
Comment (easeway) [inline .sandbox/manifest.yaml:8]: Please use `0.0.0.0:3000` for listening on all hosts.

File: mysite/settings.py [hunk 2]
```diff
@@ -25,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0', 'localhost']
```
Comment (easeway) [inline mysite/settings.py:29]: I'd recommend `ALLOWED_HOSTS = ['*']` for accepting connections from any interfaces.

**Review by easeway — APPROVED** [review]

File: .sandbox/manifest.yaml (hunk 1 above)
Comment (assefamaru) [reply .sandbox/manifest.yaml]: Good point. Probably clearer that way. Done.

File: mysite/settings.py (hunk 2 above)
Comment (assefamaru) [reply mysite/settings.py]: Updated.

**Pull request merged**